tests/pytest_framework/src/syslog_ng/syslog_ng_cli.py

```diff
@@ -112,3 +112,15 @@ def stop(self, unexpected_messages=None):
     # Helper functions
     def __error_handling(self):
         self.__console_log_reader.dump_stderr()
+        self.__handle_core_file(self.__process)
+
+    def __handle_core_file(self, process):
```

Comment:
Maybe `process` is not needed, as it can be acquired from `self`.

Reply:
@alltilla thx for the finding, fixed it.